distribution/ddf-common/src/main/resources/etc/users.attributes

```diff
@@ -1,8 +1,8 @@
 {
     "admin" : {
-
+        "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress":"admin@localhost.local"
     },
     "localhost" : {
-
+        "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress":"localhost@localhost.local"
```

Comment:
I think we should change localhost@localhost.local to maybe system@localhost.local.  I think that will make more sense once the host name is changed to something other than localhost.